src/Service.php

```diff
@@ -162,13 +175,13 @@ public function generateClass()
       $options[\'classmap\'][$key] = $value;
     }
   }' . PHP_EOL;
-        $source .= '  $options = array_merge(' . var_export($this->config->get('soapClientOptions'), true) . ', $options);' . PHP_EOL;
+        $source .= '  $options = array_merge(' . $this->adjustArrayNotation(var_export($this->config->get('soapClientOptions'), true)) . ', $options);' . PHP_EOL;
```

Comment:
Idea/Note: If we care about code style in the generated code `var_export()` is not the optimal solution. We should consider using https://github.com/Riimu/Kit-PHPEncoder or the like.